### ISSUE TITLE
Question about loss function

### ISSUE THREAD

**Issue by jonathanventura** [juglab/n2v#101]
Why is the loss function

        loss = tf.reduce_sum(K.square(target - y_pred*mask)) / tf.reduce_sum(mask)

instead of

        loss = tf.reduce_sum(K.square(target - y_pred)*mask) / tf.reduce_sum(mask)

?

**Comment by tibuch** [comment]
Hi @jonathanventura 

We initialize `target` with zeros and only set values at the active pixels. So `target - y_pred*mask` is zero everywhere except at the active pixels.

Your suggested loss should lead to the same result and looks like the saver choice to be honest :)

**Comment by jonathanventura** [comment]
Ah, that makes sense -- thank you!
